### PR TITLE
chore: Attempt to fix intuitive actions

### PR DESCRIPTION
**Pull request by glooms**
GitHub actions are great - they just work.

### CODE REVIEW

File: .github/workflows/release-enigma-go.yml [hunk 1]
```diff
@@ -22,6 +22,8 @@ jobs:
           ref: master
           fetch-depth: 0
           token: ${{ secrets.GH_ACCESS_TOKEN}}
+      - name: Setup Git environment
+        uses: ./.github/actions/git-env-setup
```
Comment (yeshQ) [inline .github/workflows/release-enigma-go.yml:26]: You have to add this YAML file under .github dir https://github.com/qlik-trial/qlik-cli/blob/master/.github/actions/git-env-setup/action.yml

Comment (yeshQ) [reply .github/workflows/release-enigma-go.yml:26]: uses command is used to import Github action.

Comment (glooms) [reply .github/workflows/release-enigma-go.yml]: Thanks! 🙇

**Review by yeshQ — APPROVED** [review]
LGTM 👍

**Pull request merged**